autoprotocol/builders.py

```diff
@@ -1542,12 +1548,30 @@ def device_mode_params(
 
         Parameters
         ----------
-        model : string, optional
-            Tempest chip model.
-        chip_material : string, optional
+        device: string
+            x_tempest_chip or x_mantis
+        model: string, optional
+            Tempest chip or mantis model.
+        chip_material: string, optional
             Material that the tempest chip is made of.
-        nozzle : string, optional
-            Type of chip nozzle.
+        nozzle: string, optional
+            Type of chip nozzle for tempest.
+        diaphragm: Optional[Integer]
+            any integer between 0 and 100, inclusive
+        nozzle_size: Optional[Unit[Length]]
+            one of "0.1:mm", "0.2:mm", "0.5:mm"
+        tubing: Optional[String]
+            one of "LV", "HV", "P200", "P1000"
+            TODO: How do we want to handle these tip types?
+            In the future, we're going to have tempest source support
+            this new tip type container type which will get a shortname
+            akin to `pipette-tip-p200` or something. If we align these
+            two values to that shortname, we can just pass it in via
+            `source_container.container_type.shortname`
+        z_drop: Optional[Unit[Length]]
+            within the range: 0:mm - 100:mm, inclusive
+        viscosity: Optional[String]
```

Comment:
this is a super nit, but the type descriptions for these are a different style than the ones above

ie, `nozzle: string, optional` vs `diaphragm: Optional[Integer]`

we should align on one format for all

Reply:
on it!